7_react/react2/week1/github_user_searcher/src/components/SearchInput.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import { GithubSearchContext } from "../context/GithubSearchContext";
+
+const SearchInput = () => {
+  const { query, setQuery, fetchUsers } = useContext(GithubSearchContext);
+
+  const handleInputChange = (event) => {
+    setQuery(event.target.value);
+  };
+
+  const handleSearch = () => {
+    fetchUsers();
+  };
+
+  return (
+    <div>
+      <input
```

Comment:
Good job! This is just a "pro tip" :D if you wrap the input by form element it also allows you to use keys such as "enter" to search

example:
```
  const handleSearch = (e) => {
    e.preventDefault();
    fetchUsers();
  };

  return (
    <form onSubmit={handleSearch}>
```